app/views/payment-links/information.njk

```diff
@@ -31,20 +35,28 @@ Create a payment link - {{currentService.name}} {{currentGatewayAccount.full_typ
     {% endif %}
     {% set confirmationLabel %}{{friendlyURL}}/{{currentService.name | removeIndefiniteArticles | slugify}}/{% endset %}
 
+    {% if isWelsh %}
+      {% set titleLabel = 'Welsh title' %}
+      {% set titleHint = 'Briefly describe what the user is paying for. For example, “Talu am drwydded barcio”. This will also be your website address.' %}
```

Comment:
Can we make sure the Welsh text is marked up in a `<span lang="cy">` element? I think using `html` rather than `text` in the `govukInput` macro will allow the `titleHint` to contain HTML (see https://design-system.service.gov.uk/components/text-input/#options-text-input-example--hint for details).

Reply:
Ah yes! I'll do that.